packages/lib/src/styles.css

```diff
@@ -1,2 +1,2 @@
 /* Global lib styles for demo/Storybook */
```

Comment:
Actually, this file is only imported by `packages/app/src/styles.css` so I guess the comment should be updated ?